client/app/vps/upgrade/vps-upgrade.html

```diff
@@ -1,107 +1,100 @@
-<section class="container-fluid cui-page__content cui-page__content-flex">
+<section class="cui-page__content cui-page__content-flex vps-upgrade">
+    <oui-back-button data-title="{{ 'vps_configuration_upgradevps_title' | translate }}"
+        data-on-click="$ctrl.cancel()"></oui-back-button>
```

Comment:
You should prefer href usage

Reply:
I cannot because there's a function beside who compute what was the previous state. I cannot guess it by my own there !